pkg/cli/common/http_status.go

```diff
@@ -25,12 +24,6 @@ func CheckResponseStatus(resp *http.Response, name string) error {
 		return NewRetryableError("too many request")
 	case resp.StatusCode == http.StatusUnauthorized:
 		return fmt.Errorf("unauthorized, please check the validity of the token")
-	case resp.StatusCode == http.StatusNotFound:
-		if name == "" {
-			return errors.New("not found")
-		}
-
-		return fmt.Errorf("%s not found", name)
```

Comment:
compared to `unexpected status code 404, ....`, `... not found` is clearer, is it possible to display something like `environment x not found`, `project x not found`?

Reply:
Update to unify the error message.